### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by dragondaddy** [comment]
Woof. That's bad. I was able to make an install on my server, I'll have a look right now.

**Comment by dragondaddy** [comment]
!testme

**Comment by dragondaddy** [comment]
!testme

**Comment by dragondaddy** [comment]
At this point I'm able to install Mitra on my home server.

BUT

After the install, I need to comment the following line in /lib/systemd/system/mitra.service:
`ProtectHome=true`

Then I need to run the following:
```
systemctl daemon-reload
systemctl start mitra
```

We're far from perfect. And I wouldn't say "nearly there". But it's something.

Also, as it is, the remove script does not get rif of the mitra deb package, you have to do it manually. More work to come.

**Comment by dragondaddy** [comment]
Trying a new approach in https://github.com/YunoHost-Apps/mitra_ynh/tree/src-install

**Comment by dragondaddy** [comment]
!testme

**Comment by dragondaddy** [comment]
!testme

**Comment by dragondaddy** [comment]
!testme

**Comment by dragondaddy** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -69,7 +69,7 @@ ynh_exec_as_app mitractl create-account $admin $adminpass admin
 # SETUP & START SERVICE
 #=================================================
 
-ynh_config_add_systemd --template=mitra
+ynh_config_add_systemd --template=mitra.service
```
Comment (ericgaspar) [inline scripts/install]: `systemd.service` will be renamed `mitra.service` by default, so `--template=mitra.service` is not needed

Comment (dragondaddy) [reply scripts/install]: Yeah, now there's no more template specified https://github.com/YunoHost-Apps/mitra_ynh/blob/bdcc96bdef8f933cce0ad1508118067cdef09957/scripts/install#L78C1-L78C23

**Pull request merged**